sdc/functions/numpy_like.py

```diff
@@ -97,3 +97,27 @@ def sdc_astype_number_impl(self, dtype):
             return arr
 
         return sdc_astype_number_impl
+
+
+@sdc_register_jitable
+def sum(a):
+    length = len(a)
+    result = 0
+    for i in prange(length):
+        if not numpy.isnan(a[i]):
```

Comment:
Let's use here `get_isnan` @PokhodenkoSA  knows the details

Reply:
I think @AlexanderKalistratov means this https://github.com/IntelPython/sdc/pull/579#discussion_r376252981